app/main.py

```diff
@@ -1,3 +1,62 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
```

Comment:
"km" can be int also, so in others annotayions change it too

Reply:
You are not quite right! When we implement the magic method __truediv__ - it returns an instance of the class that takes as an argument the result of division of two numbers, which is more often of type float.

Reply:
> "km" can be int also, so in others annotayions change it too


Please fix the above request